ios/RNSScreenStack.mm

```diff
@@ -1,21 +1,49 @@
-#import "RNSScreenStack.h"
-#import "RNSScreen.h"
-#import "RNSScreenStackAnimator.h"
+#ifdef RN_FABRIC_ENABLED
+// TODO: merge this import when StackHeaderConfigComponentView is merged
+#import "RNSScreenStackHeaderConfigComponentView.h"
+
+#import <React/RCTMountingTransactionObserving.h>
+
+#import <React/UIView+React.h>
+#import <react/renderer/components/rnscreens/ComponentDescriptors.h>
+#import <react/renderer/components/rnscreens/EventEmitters.h>
+#import <react/renderer/components/rnscreens/Props.h>
+#import <react/renderer/components/rnscreens/RCTComponentViewHelpers.h>
+
+#import "RCTFabricComponentsPlugins.h"
+
+#import <React/RCTSurfaceTouchHandler.h>
+#else
 #import "RNSScreenStackHeaderConfig.h"
-#import "RNSScreenWindowTraits.h"
 
 #import <React/RCTBridge.h>
 #import <React/RCTRootContentView.h>
 #import <React/RCTShadowView.h>
 #import <React/RCTTouchHandler.h>
 #import <React/RCTUIManager.h>
 #import <React/RCTUIManagerUtils.h>
+#endif // RN_FABRIC_ENABLED
 
-@interface RNSScreenStackView () <
-    UINavigationControllerDelegate,
-    UIAdaptivePresentationControllerDelegate,
-    UIGestureRecognizerDelegate,
-    UIViewControllerTransitioningDelegate>
+#import "RNSScreen.h"
+#import "RNSScreenStack.h"
+#import "RNSScreenStackAnimator.h"
+#import "RNSScreenWindowTraits.h"
+
+@interface RNSScreenStackView ()
+#ifdef RN_FABRIC_ENABLED
+    <UINavigationControllerDelegate,
+     UIAdaptivePresentationControllerDelegate,
+     UIGestureRecognizerDelegate,
+     UIViewControllerTransitioningDelegate,
+     RCTMountingTransactionObserving> {
+  BOOL _updateScheduled;
```

Comment:
I think `_updateScheduled` is not needed here since it can be added in `@implementation` and we can just add `#ifdef RN_FABRIC_ENABLED` on `RCTMountingTransactionObserving` not to duplicate other protocols.